ngraph/test/op_eval/floor_mod.cpp

```diff
@@ -5,35 +5,87 @@
 #include <string>
 #include <vector>
 
-#include "gtest/gtest.h"
-
 #include "ngraph/op/floor_mod.hpp"
-#include "ngraph/runtime/host_tensor.hpp"
-#include "ngraph/validation_util.hpp"
-#include "runtime/backend.hpp"
-#include "util/test_tools.hpp"
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/engine/interpreter_engine.hpp"
+#include "util/engine/test_engines.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
 
 using namespace std;
 using namespace ngraph;
 
+static string s_manifest = "${MANIFEST}";
```

Comment:
Is this variable used anywhere?

Reply:
It is not. I didn't notice that there is a TEST macro instead of NGRAPH_TEST